scripts/tools/zap/version_update.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Do we need a hashbang at all?

Reply:
Well, without it running it just as an executable doesn't really work, right?